src/VisualStudio/IntegrationTest/TestUtilities/VisualStudioInstance.cs

```diff
@@ -110,6 +112,15 @@ public VisualStudioInstance(Process hostProcess, DTE dte, ImmutableHashSet<strin
                 }
             }
 
+            using var timer = new System.Threading.Timer(_ =>
+            {
+                var procDumpInfo = ProcDumpInfo.ReadFromEnvironment();
+                if (procDumpInfo != null)
+                {
+                    ProcDumpUtil.CreateCrashDump(procDumpInfo.Value, hostProcess.Id);
+                }
+            }, null, TimeSpan.FromMinutes(10), period: Timeout.InfiniteTimeSpan);
```

Comment:
💭 I'm guessing this part is temporary, and the proper PR here will just rely on `--blame-hang`?

Reply:
Yeah, going to remove this now. There were two theories as to why we weren't getting the dumps from the standard system:

1. The existing procdump instance was interfering.
2. Whatever logic dotnet test had for making it's own dumps didn't work for devenv for some reason.

This was hedging if it was the latter reason.